behovsmeldingsmodell/build.gradle.kts

```diff
@@ -7,6 +7,16 @@ group = "no.nav.hjelpemidler"
 version = System.getenv("VERSION_TAG") ?: "local"
 
 dependencies {
+    /*
+     TODO: Det tryggeste er å ha denne avhengigheten her, slik at vi kan sikre at den alltid blir brukt,
+      men samtidig vi vil helst ikke ha ekstra avhengigheter her. Kan vi gjøre det på en bedre måte?
+     */
+
+    // TODO: legg til i hm-katalog
+
+    // OWASP
+    implementation("com.googlecode.owasp-java-html-sanitizer:owasp-java-html-sanitizer:20240325.1")
```

Comment:
Denne er for å sikre at tekstene ikke inneholder noe annet enn `<em/>` og `<strong/>`, slik at vi kan få med formatering. Frontend kan ha en egen sanitation i tillegg, men pdfgen må få ferdig sanitized fra hm-soknad-api.